docs/source/refs.bib

```diff
@@ -1,24 +1,9 @@
-@inproceedings{wille2019towardsHDLsynthesis,
-    author = {Wille, Robert and Haghparast, Majid and Adarsh, Smaran and M, Tanmay},
-    title = {Towards {HDL}-based Synthesis of Reversible Circuits with No Additional Lines},
-    booktitle = {International Conference on Computer Aided Design},
-    year = {2019},
-    url = {https://www.cda.cit.tum.de/files/eda/2019_iccad_hdl_based_reversible_circuit_synthesis_without_additional_lines.pdf},
-}
-
-@inproceedings{wille2010syrec,
-  author={Wille, Robert and Offermann, Sebastian and Drechsler, Rolf},
-  title={{SyReC}: A programming language for synthesis of reversible circuits},
-  booktitle={Forum on Specification \& Design Languages},
-  year={2010},
-  url = {http://www.informatik.uni-bremen.de/agra/doc/konf/10_syrec_reversible_hardware_language.pdf},
-}
-
-@article{wille2016syrec,
-  author={Wille, Robert and Sch{\"o}nborn, Eleonora and Soeken, Mathias and Drechsler, Rolf},
-  title={{SyReC}: A hardware description language for the specification and synthesis of reversible circuits},
-  journal={Integration},
-  year={2016},
-  publisher={Elsevier},
-  url = {https://www.sciencedirect.com/science/article/abs/pii/S016792601500125X?via%3Dihub},
+@article{ADARSH2022100451,
```

Comment:
Please use a consistent naming scheme for the reference keys, i.e., follow the style of the existing references.